templates/tomcat/tomcat.conf.erb

```diff
@@ -0,0 +1,51 @@
+# This is managed by Puppet, DO NOT EDIT!
+#
+# System-wide configuration file for tomcat services
+# This will be loaded by systemd as an environment file,
+# so please keep the syntax. For shell expansion support
+#
+# There are 2 "classes" of startup behavior in this package.
+# The old one, the default service named tomcat.service.
+# The new named instances are called tomcat@instance.service.
+#
+# Use this file to change default values for all services.
+# Change the service specific ones to affect only one service.
+# For tomcat.service it's /etc/sysconfig/tomcat, for
+# tomcat@instance it's /etc/sysconfig/tomcat@instance.
+
+# This variable is used to figure out if config is loaded or not.
+TOMCAT_CFG_LOADED="<%= scope.lookupvar('::candlepin::tomcat_cfg_loaded') %>"
```

Comment:
When would a user override this?

Reply:
I have no idea, i just copied in the default tomcat.conf

Reply:
They wouldn’t, it’s used by tomcat so it know if it’s loaded it own config file. The default comment isn’t clear imo. 

I’d say if we can’t see anyway anyone would want to change it let’s just make it possible to override via hiera and not an installer option.

Reply:
Do you have an example of how to do that?

Reply:
@sean797 the installer doesn't expose the candlepin module directly so that's not an issue.

@chris1984 if the variable is not expected to be overridden/changed then I'd hardcode it.